domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -922,6 +933,52 @@ public void save(AsyncResult<SQLConnection> sqlConnection, String table, String
     }
   }
 
+  /**
+   * Save entity in table and return the updated version (after database trigger for example).
```

Comment:
can you change "and return the updated version (after database trigger for example)." to "and return the updated entity."? "version" is ambiguous, and the details should go into the `@param replyHandler` javadoc.